Dockerfile

```diff
@@ -23,4 +23,4 @@ ADD .travis.yml .
 ADD .rubocop.yml .
 ADD test /app/test
 
-CMD bundle exec bin/secrets
+CMD mkdir -p /secrets/bin && cp /app/bin/wait_for_it.sh /secrets/bin && bundle exec bin/secrets
```

Comment:
doesn't work in a run-statement, I suspect some kind of race in k8s as /secrets is a file-system that it mounts.